config/config.go

```diff
@@ -17,6 +18,9 @@ type NodeConfig interface {
 
 	// benchmark consensus
 	BenchmarkConsensusRoundRetryIntervalMillisec() uint32
+
+	// block storage
+	BlockSyncCommitTimeout() time.Duration
```

Comment:
Let's change the function name to be consistent with the variable, like `BlockSyncCommitTimeoutMS()`